jax/_src/numpy/fft.py

```diff
@@ -252,17 +252,172 @@ def ifftn(a: ArrayLike, s: Shape | None = None,
+  """Compute a multidimensional discrete Fourier transform of a real-valued array.
+
+  JAX implementation of :func:`numpy.fft.rfftn`.
+
+  Args:
+    a: real-valued input array.
+    s: optional sequence of integers. Specifies the effective size of the output
+      along each specified axis. If not specified, it will default to the
+      dimension of input along ``axes``.
+    axes: optional sequence of integers, default=None. Specifies the axes along
+      which the transform is computed.
+    norm: string, default="backward". The normalization mode. "backward", "ortho"
+      and "forward" are supported.
+
+  Returns:
+    An array containing the multidimensional discrete Fourier transform of ``a``
+    having size specified in ``s`` along the axes ``axes`` except along the axis
+    ``axes[-1]``. The size of the output along the axis ``axes[-1]`` is 
+    ``(s[-1]/2)+1``, if ``s[-1]`` is even and ``(s[-1]+1)/2``, if ``s[-1]`` is odd.
+
+  See also:
+    - :func:`jax.numpy.fft.rfft`: Computes a one-dimensional discrete Fourier
+      transform of real-valued array.
+    - :func:`jax.numpy.fft.rfft2`: Computes a two-dimensional discrete Fourier
+      transform of real-valued array.
+    - :func:`jax.numpy.fft.irfftn`: Computes a real-valued multidimensional inverse
+      discrete Fourier transform.
+
+  Examples:
+    ``jnp.fft.rfftn`` computes the transform along all the axes by default.
+
+    >>> x = jnp.array([[[1, 3, 5],
+    ...                 [2, 4, 6]],
+    ...                [[7, 9, 11],
+    ...                 [8, 10, 12]]])
+    >>> with jnp.printoptions(precision=2, suppress=True):
+    ...   jnp.fft.rfftn(x)
+    Array([[[ 78.+0.j  , -12.+6.93j],
+            [ -6.+0.j  ,   0.+0.j  ]],
+    <BLANKLINE>
+           [[-36.+0.j  ,   0.+0.j  ],
+            [  0.+0.j  ,   0.+0.j  ]]], dtype=complex64)
+
+    When ``s=[3, 3, 4]``, transform is compted along ``axes (-3, -2, -1)``
+    dimension of the transform along ``axes -3 and -2`` will be ``3 and 3``,
+    along ``axis -1`` will be ``(4/2)+1) = 3`` and dimension along other axes
+    (if available) will be the same as that of input.
+
+    >>> with jnp.printoptions(precision=2, suppress=True):
+    ...   jnp.fft.rfftn(x, s=[3, 3, 4])
+    Array([[[ 78.   +0.j  , -16.  -26.j  ,  26.   +0.j  ],
+            [ 15.  -36.37j, -16.12 +1.93j,   5.  -12.12j],
+            [ 15.  +36.37j,   8.12-11.93j,   5.  +12.12j]],
+    <BLANKLINE>
+           [[ -7.5 -49.36j, -20.45 +9.43j,  -2.5 -16.45j],
+            [-25.5  -7.79j,  -0.6 +11.96j,  -8.5  -2.6j ],
+            [ 19.5 -12.99j,  -8.33 -6.5j ,   6.5  -4.33j]],
+    <BLANKLINE>
+           [[ -7.5 +49.36j,  12.45 -4.43j,  -2.5 +16.45j],
+            [ 19.5 +12.99j,   0.33 -6.5j ,   6.5  +4.33j],
+            [-25.5  +7.79j,   4.6  +5.04j,  -8.5  +2.6j ]]], dtype=complex64)
+
+    When ``s=[3, 5]`` and ``axes=(0, 1)``, transform is computed along
+    ``axes (0, 1)`` and size of the transform along ``axis 0`` will be ``3``,
```

Comment:
Remove "transform is computed along axex (0, 1)": seems somewhat repetetive

Reply:
Removed